### PR TITLE
unit tests for un-tested files

### PR DESCRIPTION
**Pull request by bogdan-rosianu**
Improved code coverage for different packages / files.
No system tests needed

### CODE REVIEW

File: core/dblookupext/factory/historyRepositoryFactory_test.go [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+	hrf, err = factory.NewHistoryRepositoryFactory(argsNilHasher)
+	require.Equal(t, core.ErrNilHasher, err)
+	require.Nil(t, hrf)
+
+	hrf, err = factory.NewHistoryRepositoryFactory(args)
+	require.NoError(t, err)
+	require.False(t, check.IfNil(hrf))
+}
+
+func TestHistoryRepositoryFactory_Create_ShouldCreateDisabledRepository(t *testing.T) {
```
Comment (iulianpascalau) [inline core/dblookupext/factory/historyRepositoryFactory_test.go:42]: Let's not use double _ as to respect the pattern on all other tests

**Review by iulianpascalau — COMMENTED** [review]
We still need to add tests for untested view functions. Check Jira tasks.

**Review by iulianpascalau — COMMENTED** [review]
Please add description

File: core/dblookupext/factory/historyRepositoryFactory_test.go (hunk 1 above)
Comment (sasurobert) [reply core/dblookupext/factory/historyRepositoryFactory_test.go:42]: +1

Comment (bogdan-rosianu) [reply core/dblookupext/factory/historyRepositoryFactory_test.go]: ok, removed

**Review by sasurobert — APPROVED** [review]

**Review by iulianpascalau — APPROVED** [review]

**Review by LucianMincu — APPROVED** [review]
👍

**Pull request merged**